.github/workflows/ci_cron_daily.yml

```
@@ -4,6 +4,14 @@ on:
   schedule:
     # run every day at 3am UTC
     - cron: '0 3 * * *'
+  pull_request:
+    # We also want this workflow triggered if the 'Extra CI' label is added
+    # The first three are the defaults
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
```

Isn't just watching `labeled` enough if you want the label to trigger it?
```suggestion
      - labeled
```

---

Well you probably don't need opened, but syncronize and reopened are needed so they still run on subsequent commits.

---

To be clear the first 3 are the default behaviour of `pull_request, so as written this is adding labeled.

---

Well, then let's keep the others but remove `opened`.

---

Sure, it doesn't really matter though, it just doesn't run.

---

Slightly more eco-friendly if we just not have it start in the first place if not necessary?